src/Compilers/CSharp/Test/Symbol/Symbols/RequiredMembersTests.cs

```diff
@@ -1265,6 +1273,35 @@ public class C
         comp.VerifyDiagnostics(expectedDiagnostics);
     }
 
+    [Theory]
+    [CombinatorialData]
+    public void EnforcedRequiredMembers_NoInheritance_NoneSet_HasSetsRequiredMembers(bool useMetadataReference, [CombinatorialValues("", " C")] string constructor, [CombinatorialValues("", "method: ")] string target)
+    {
+#pragma warning disable IDE0055 // Fix formatting
```

Comment:
nit: is there an issue tracking this? I assume it's just a matter of updating the CodeStyle analyzer that we reference.

Reply:
Let's open an issue and track removing those

Reply:
Will be fixed by https://github.com/dotnet/roslyn/pull/60655